examples/src/indexes/vector_stores/analyticdb.ts

```diff
@@ -0,0 +1,51 @@
+import { AnalyticDBVectorStore } from "langchain/vectorstores/analyticdb";
+import { OpenAIEmbeddings } from "langchain/embeddings/openai";
+
+export const run = async () => {
+  const connectionOptions = {
+    host: process.env.PG_HOST || "localhost",
```

Comment:
nit: since some of our other examples automatically read from environment variables, prefixing with `ANALYTICDB_` would be nicer here to avoid conflicts (e.g. `ANALYTICDB_PG_HOST`)